MAINTAINERS-example

```diff
@@ -1,33 +1,23 @@
-# This project, topsikrit, has top-secret awesome sauce.
-#
-# Btw, note that this comment was human-edited and will be preserved
-# even if you add/remove maintainers using the gem. Nifty!
-
-maintainers: {
-    version: 1
-    issues: "https://github.com/csprojects/topsikrit/issues"
```

Comment:
My thinking too. The most recent commit here adds a schema which assumes that `issues` is mandatory. There will be some churn here for project 200, so it would be nice to be able to programmatically search the repos for given issues urls.